client/src/languagePlugins/javascript/javascriptPlugin.ts

```diff
@@ -209,6 +214,32 @@ class JavascriptBlockKind implements Langs.Block {
   export const javascriptLanguagePlugin : Langs.LanguagePlugin = {
     language: "javascript",
     editor: javascriptEditor,
+    evaluate: (scopeDictionary:{}, node:Graph.Node) => {
```

Comment:
We should not need to pass `scopeDictionary` to the `evaluate` function. Instead, we can get the variable values from the `JsCodeNode`.

Reply:
Have removed scope dictionary.